.github/workflows/style.yml

```diff
@@ -31,15 +31,15 @@ jobs:
         uses: actions/setup-python@v5
         if: steps.changed-files.outputs.any_changed == 'true'
         with:
-          python-version: '3.10'
+          python-version: "3.10"
```

Comment:
Unnecessary changes.

Reply:
Oops! My linter caused that. I've changed it back.